cmd/better-retester/retester.go

```diff
@@ -55,27 +55,19 @@ func (c *retestController) sync() error {
 		return err
 	}
 
-	rawPools, err := c.dividePool(candidates)
-	if err != nil {
-		return err
+	logrus.Infof("Found %d candidates for retest (pass label criteria, fail some tests)", len(candidates))
```

Comment:
Do we know "fail some tests" here?

Reply:
Yeah - that's something I added to the `query` method: